client/components/main/style.scss

```diff
@@ -17,3 +17,8 @@
 .main .empty-content {
 	margin-top: 0;
 }
+
+// Use Recoleta on all page headings
+.main .formatted-header__title {
```

Comment:
Still too inclusive, how about:
```
.current-section + .formatted-header .formatted-header__title
```

Note that home has an extra container around the header so we might need to tweak this a bit.

Reply:
Tried with a different selector in 951b112 that should be specific for page headings only. Needed to tweak a couple of pages in order to make the header a direct child of the main component, but shouldn't change the layout. 

>Note that home has an extra container around the header so we might need to tweak this a bit.
Yeah, that was tricky. I just left the container and made sure the font is used with a explicit font family definition in the home styles.